.codespell/ignore_lines.txt

```diff
@@ -0,0 +1,36 @@
+;; SPDX-FileCopyrightText = "2009 Fermi Research Alliance, LLC"
+;; SPDX-License-Identifier = "Apache-2.0"
+;; .github/issue-labeler-config-regexes.yaml
+  - 'Components: *(?:The affected component,? (?:due to this bug|if any, from this feature) *)?\[?.*(\b[Rr]elease\b)'
+  - 'Components: *(?:The affected component,? (?:due to this bug|if any, from this feature) *)?\[?((?:\b(?!The affected|[Ff]rontend [Mm]onitoring|[Ff]actory|[Gg]lidein|[Dd]ocumentation|(CI|ci|[Tt]esting)|[Rr]elease|[Ff]actory [Mm]onitoring|[Ff]rontend)\b)[^, \n\]]+)\]?'
+;; creation/web_base/condor_platform_select.sh
+        "$error_gen" -error "condor_platform_select.sh" "Config" "$STR" "SupportAutodetect" "False" "ArchType" "Unknown"
+;; creation/web_base/singularity_lib.sh
+OSG_SINGULARITY_BINARY_DEFAULT="/cvmfs/oasis.opensciencegrid.org/mis/singularity/bin/singularity"
+[[ -n "$CVMFS_MOUNT_DIR" ]] && OSG_SINGULARITY_BINARY_DEFAULT="${CVMFS_MOUNT_DIR}/oasis.opensciencegrid.org/mis/singularity/bin/singularity"
+;; doc/factory/monitoring.html
+                                            Run    Held    Idle  Unknwn | Pending    Wait   StgIn  StgOut |  RunDiff IdleDiff   %RD
+frontend_vofrontend_service_frontend        Run    Held    Idle  Unknwn | Pending    Wait   StgIn  StgOut |  RunDiff IdleDiff   %RD
+Unknwn : Status Unknown (StatusIdleOther)
```

Comment:
Should `Unknwn` be changed to `Unknown`?

Reply:
No, I added it here because codespell was changing it. Condor spelling